netlify.toml

```diff
@@ -33,3 +33,8 @@ to = "/docs/v3.4.0/reporting_bugs"
 status = 301
 force = false
 
+[[redirects]]
+from = "/blog/jepsen-343-results"
+to = "/blog/2020/01/30/latest-jepsen-results-against-etcd-3.4.3/"
+status = 301
+force = false
```

Comment:
Why this redirect?

Reply:
I don't want to break any potential bookmarks -- so i'm trying to do the conversion without changing any links. I'm adding this one specifically because docsy adds a different url style for the blog that i think is better, so i added the redirect for the blog entry. (This is helped by the fact that we only have one, so there's not a lot of redirecting to do...)

Reply:
Oh, the blog style doesn't need to be that way AFAIK. We can look into that later.

Reply:
I know we can change it -- I set it to the way I thought best :)
Let's open an issue for discussion? I have some pages that I'm expecting to make in to blog entries, so it'll be good to get that sorted.